App.js

```diff
@@ -1,26 +1,70 @@
-import React from 'react'
-import styled from 'styled-components/native'
+import React, { useState } from 'react';
+import styled from 'styled-components/native';
+
+import CharacterRandomizer from './CharacterRandomizer';
 
 const Container = styled.View`
-  flex: 1;
-  background-color: papayawhip;
-  justify-content: center;
-  align-items: center;
-`
+	flex: 1;
+	background-color: #443423;
+	justify-content: center;
+	align-items: center;
+`;
 
 const Title = styled.Text`
-  font-size: 24px;
-  color: palevioletred;
-`
+	font-size: 24px;
+	color: #dbc3a0;
+	margin-top: 50px;
+`;
+
+const StyledImage = styled.Image`
+	width: 300px;
+	height: 300px;
+	margin: 10px;
+	border-radius: 12px;
+`;
+
+const StyledButton = styled.Button`
+	font-size: 24px;
+	color: palevioletred;
+	background-color: #dbc3a0;
+	margin: 10px;
+`;
+
+const image = { uri: 'https://picsum.photos/id/1028/200/300' };
 
 const App = () => {
-  return (
-    <Container>
-      <Title>This is your cool app!</Title>
-      <Title>Go to App.js and start coding</Title>
-      <Title>💅💅💅</Title>
-    </Container>
-  )
-}
-
-export default App
+	const [showRandomCharacter, setShowRandomCharacter] = useState(false);
+	const [hideWelcomeScreen, setHideWelcomeScreen] = useState(false);
```

Comment:
From naming point of view, those two state properties shouldn't have possible state value in their names (show, hide), but rather information about what they do. So instead of :

```
[showRandomCharacter, setShowRandomCharacter]
[hideWelcomeScreen, setHideWelcomeScreen]
```

they should rather be named :

```
[randomCharacterVisibility, setRandomCharacterVisibility]
[welcomeScreenVisibility, setWelcomeScreenVisibility]
```